app/src/main/res/layout/nav_header_main.xml

```diff
@@ -12,14 +12,6 @@
     android:paddingBottom="@dimen/activity_vertical_margin"
     android:theme="@style/ThemeOverlay.AppCompat.Dark">
 
-    <ImageView
```

Comment:
M-am jucat eu putin cu designul la meniu, nu bagati in seama, oricum schimbam

Reply:
![image](https://user-images.githubusercontent.com/18668593/68817655-00b05b80-068b-11ea-924f-9514843b68be.png)

![image](https://user-images.githubusercontent.com/18668593/68817665-0b6af080-068b-11ea-9b7e-b7a0065cdb32.png)


Cum arata requestul care il fac pe on click. Putem da notificari asa si in postman etc